### DISCUSSION TITLE
Does nobody find SSL useful for remote access?

### DISCUSSION THREAD

**Discussion by Tebrion74** [openbroadcaster/obplayer#45] [Ideas]
SSL simply does not work. I have valid certificates and I cannot find any documentation on this and a response from support said its the only 'feature' that does nothing. 

Can we get an ETA on this? This is a deal breaker for this software for me... looks like it back to clunky azuracast.

**Comment by coryshort** [comment]
ssl should be used at all times now seeing as some browsers will now block non ssl items

**Comment by allexandry** [comment]
hello, 
have u found solution?

**Comment by Tebrion74** [comment]
yes ... find sales team

**Reply by radiorob** [reply]
Ha ha, we don't have a software sales team, all our source code is public.  We do accept support from Sponsors for new features and development.  I totally agree SSL is important, use letsencrypt. I have seen it work with the player, but not documented enough to get into main code base.
